### PR TITLE
Prevent a panic if no host config is found with a nicer error message

### PR DESCRIPTION
**Pull request by MiguelMoll**
New output would look like:

```
convox apps
ERROR: no host config found, try `convox login`
```

### CODE REVIEW

**Comment by MiguelMoll** [comment]
[Rack segfault when not logged in](https://trello.com/c/PZXakHrK/2099-rack-segfault-when-not-logged-in)

**Comment by MiguelMoll** [comment]
[Rack segfault when not logged in](https://trello.com/c/PZXakHrK/2099-rack-segfault-when-not-logged-in)

File: cmd/convox/main.go [hunk 1]
```diff
@@ -112,12 +112,11 @@ func currentRack(c *cli.Context) string {
 func rackClient(c *cli.Context) *client.Client {
 	host, password, err := currentLogin()
 	if err != nil {
-		stdcli.Error(err)
-		return nil
```
Comment (awsmsrc) [inline cmd/convox/main.go]: i get why this was changed like this, that function is called everywhere.. Should it just return an error "the go way?"

**Comment by awsmsrc** [comment]
I've added a comment, I can see how this fixes the problem. 

I'm ok with the change but it isn't the standard "the go way"

File: cmd/convox/main.go (hunk 1 above)
Comment (MiguelMoll) [inline cmd/convox/main.go]: At first, I went with this function returning an error. But as you mentioned, it's called everywhere in a chained fashion. All those calls would have to add error checking, which is fine in itself but then reduces the usefulness of this helper IMHO.

So I went with the function exiting out as the CLI can't do anything without a working client.

**Comment by MiguelMoll** [comment]
Fixed via https://github.com/convox/rack/pull/2074

**Pull request closed without merging**